tests/PCA.py

```diff
@@ -0,0 +1,60 @@
+import pandas as pd
+import numpy as np
+from sklearn.decomposition import PCA
+from sklearn.preprocessing import StandardScaler
+import matplotlib
+import matplotlib.pyplot as plt
+
+
+# Reading the data
+
+num_components = 2
+#importing 1112X9 data frame
+path1 = "1112x9.csv"
+#importing 17X9 data frame
+path2 = "17x9.csv"
+df1 = pd.read_csv(path1)
+df2 = pd.read_csv(path2)
+
+#defining the function for PCA
+
+def MyPCA(X):
```

Comment:
You can remove the "My" from the function name. Also, please have a more descriptive name for the function argument and use type annotations.

```suggestion
def PCA(dataframe: pd.DataFrame) -> pd.DataFrame:
```

Reply:
done now!